{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/users/adapters.py

```diff
@@ -14,3 +14,13 @@ def is_open_for_signup(self, request: HttpRequest):
 class SocialAccountAdapter(DefaultSocialAccountAdapter):
     def is_open_for_signup(self, request: HttpRequest, sociallogin: Any):
         return getattr(settings, "ACCOUNT_ALLOW_REGISTRATION", True)
+
+    def populate_user(self, request, sociallogin, data):
+        user = sociallogin.user
+        if name := data.get("name"):
+            user.name = name
+        elif first_name := data.get("first_name"):
+            user.name = first_name
+            if last_name := data.get("last_name"):
+                user.name += f" {last_name}"
```

Comment:
I seems like it would be more consistent with django-allauth if we were to do it in the `save_user()` method: https://github.com/pennersr/django-allauth/blob/7060e3fb6e00f01bd881890546244670929b5cf3/allauth/account/adapter.py#L224-L251

Unless there a specific reason not to?

Reply:
It's been awhile, but I believe `save_user` isn't actually used for social registration (i.e. it's not called)